src/mmw/js/src/core/weatherStationLayer.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+var L = require('leaflet'),
+    Backbone = require('../../shim/backbone'),
+    Marionette = require('../../shim/backbone.marionette'),
+    weatherStationPopupTmpl = require('./templates/weatherStationPopup.html');
+
+var Layer = {
+    createLayer: function(geojsonFeatureCollection) {
+        return L.geoJson(JSON.parse(geojsonFeatureCollection), {
+            pointToLayer: function (feature, latlng) {
+                return L.circleMarker(latlng, {
+                    fill: true,
+                    fillColor: 'steelblue',
+                    fillOpacity: 0.2
+                });
+            },
+
+            onEachFeature: function(feature, marker) {
+                var model = new Backbone.Model(feature.properties),
+                    view = new WeatherStationPopupView({ model: model });
+
+                marker.bindPopup(view.render().el, { className: 'data-catalog-popover'});
+            },
+        });
+    }
+};
+
+var WeatherStationPopupView = Marionette.ItemView.extend({
+    template: weatherStationPopupTmpl,
+    className: 'weather-station-popup',
+
+    templateHelpers: function() {
+        return {
+            station_id: this.model.get('station_id'),
+            location: this.model.get('location'),
+            beginYear: this.model.get('beginYear'),
+            endYear: this.model.get('endYear'),
+            growStart: this.model.get('growStart'),
+            growEnd: this.model.get('growEnd'),
+            meanPrecip: this.model.get('meanPrecip')
+        };
+    }
+});
+
+module.exports.Layer = Layer;
+module.exports.WeatherStationPopupView = WeatherStationPopupView;
```

Comment:
`WeatherStationPopupView` is never used outside of this module, don't need to export it.